config/repos.yaml

```diff
@@ -219,6 +219,10 @@ Katello/katello:
   link_to_redmine: true
 Katello/katello-agent:
```

Comment:
Maybe now is a good time to remove this katello-agent portion since the repo was renamed

Reply:
Thanks. I didn't know that. Changed it into a rename now.